blocks_test.go

```diff
@@ -0,0 +1,107 @@
+// +build unit
+
+package wavelet
+
+import (
+	"crypto/rand"
+	"github.com/perlin-network/wavelet/store"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestNewRounds(t *testing.T) {
```

Comment:
Care to rename the prefixes to be Blocks instead of Rounds?

Reply:
sry, my bad